### PR TITLE
Added option to update project files

### PR DESCRIPTION
**Pull request by vsavovlime**

### CODE REVIEW

File: cli-commands/init/init.js [hunk 1]
```diff
@@ -46,24 +46,32 @@ const dockerFilesDestination = `${dockerDir}`;
...
+  
     setupDocker();
 
-    print('===== Aeproject was successfully initialized! =====');
-
-  } catch (e) {
-    printError(e.message)
-    console.error(e);
+    print('===== Aeproject was successfully updated! =====');
+  }else {
```
[the remark this answers is not in the document]
Comment (vsavovlime) [reply cli-commands/init/init.js]: It's doesn't make any sense, logically will work same and with if/else is more readable, ok ?

[the remark this answers is not in the document]
Comment (Perseverance) [reply cli-commands/init/init.js]: My 2 cents here. Having more nested code is generally less readable. When you have else you need to remember which if is this. When reading this code can sometimes lead to misunderstandings. I generally use else very rarely

**Comment by maptuhec** [comment]
@vsavovlime  Have you added unit tests for this?

File: test/commands-tests/init.js [hunk 2]
```diff
@@ -35,7 +35,43 @@ describe('Aeproject Init', () => {
 		assert.isTrue(fs.existsSync(`${executeOptions.cwd}${constants.testsFiles.dockerNginxDefault}`), "docker nginx-default doesn't exist");
 		assert.isTrue(fs.existsSync(`${executeOptions.cwd}${constants.testsFiles.dockerNginxWs}`), "docker nginx-ws doesn't exist");
 		assert.isTrue(fs.existsSync(`${executeOptions.cwd}${constants.testsFiles.dockerKeys}`), "docker keys folder doesn't exist");
+	})
+
+
+	it.only('Should update project successfully', async () => {
```
Comment (Perseverance) [inline test/commands-tests/init.js]: Pretty sure we should not have only

Comment (vsavovlime) [reply test/commands-tests/init.js]: Fixed

**Comment by maptuhec** [comment]
Should we add this to the documentation? I think we should

**Pull request merged**